src/layout/components/Page/index.vue

```diff
@@ -139,7 +139,7 @@ export default {
       clearTimeout(this.longPressTimer)
     },
     handleUpgrade() {
-      const url = 'http://www.jumpserver.org/support/'
+      const url = 'https://www.lxware.hk/pages/about'
       window.open(url, '_blank')
     }
   }
```

Comment:
There doesn't appear to be any significant coding changes between 2021 and 2025, but there is a potential issue with variable names in the export declaration which should be changed if it makes future maintenance easier. The change suggests:

* Change "export default {"

to 
```javascript
const exportDefault = {};

```

This way, when the file exports the constant as an object, it can later use the `default` property to access it without having to explicitly declare another level of nesting within the main block.

Other than this formatting suggestion, no major errors were reported. There's nothing suspicious or unexpected detected at present based solely on these snippet details.